content/en/tools-and-resources/platform-tools/log-driver.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: CDS log driver 🇨🇦
+title: CDS log driver 🇨
```

Comment:
Minor thing, but was this emoji removed intentionally?

Reply:
Thanks Bethany! Yes, was removed intentionally. Let me know if you think we should keep though!